spec/vmpooler/pool_manager_spec.rb

```diff
@@ -228,6 +228,7 @@
       allow(redis).to receive(:smembers).with('vmpooler__running__pool1').and_return([])
       allow(redis).to receive(:smembers).with('vmpooler__completed__pool1').and_return([])
       allow(redis).to receive(:smembers).with('vmpooler__discovered__pool1').and_return([])
+      allow(redis).to receive(:smembers).with('vmpooler__migrating__pool1').and_return([])
```

Comment:
I went through some months ago and converted most of our specs away from mocking/stubbing on redis, etc.  I stopped short on the pool_manager specs because there's just a lot of mess in that library and it needs deeper work (I think) to clean things up. 

I'm fine here with continuing in the existing style (because the cost of not doing so is large), even if the tests end up less than ideal: brittle, not fully exercising functionality.